internal/pkg/testproject/project.go

```diff
@@ -132,9 +135,40 @@ func (p *Project) Clear() {
 		}
 	}
 
+	p.clearSchedules()
+
 	p.logf("Test project cleared | %s", time.Since(startTime))
 }
 
+// Clear deletes all schedules.
+func (p *Project) clearSchedules() {
+	logger, _ := utils.NewDebugLogger()
+	hostName, _ := p.api.GetSchedulerApiUrl()
+	schedulerApi := scheduler.NewSchedulerApi(
```

Comment:
Je to dobre, ak by som mohol este jednu vec:
- schedulerApi by bolo dobre vytvarat hned za Storage API:
   - https://github.com/keboola/keboola-as-code/blob/4a090b0fbb2af1c6b69f9ec8165ce03626891f65/internal/pkg/testproject/project.go#L67
- `testproject` sa toti medzi testami vramci jedneho `package` znovupouziva, a tak sa usetri jeden API call, na zistenie `hostName`.